### PR TITLE
fix type of keywords entry in frontmatter (in /engine/)

### PR DESCRIPTION
**Pull request by gdevillele**
### Proposed changes

in `/engine/` :
change `keywords`'s type from array to string.

### Project version

current stable

### Related issue

contributes to #435 

Signed-off-by: Gaetan de Villele <gdevillele@gmail.com>

### CODE REVIEW

**Comment by johndmulhausen** [comment]
Deploy preview processing.

Built with commit fb118de44140afd40a76522348d2462b77319a9f

https://app.netlify.com/sites/docker-docs/deploys/582648f3d6865d75b5eef380

**Comment by johndmulhausen** [comment]
Thanks, @gdevillele !

**Comment by johndmulhausen** [comment]
Thanks, @gdevillele :)

**Comment by gdevillele** [comment]
@thaJeztah yes, thinking about it, I should have discarded those last-line-changes before committing. Is there some documentation about this for the docs case? Are these trailing new lines required on this repo?

File: engine/admin/ambassador_pattern_linking.md [hunk 1]
```diff
@@ -151,4 +150,4 @@ case `192.168.1.52:6379`.
     	apk add socat && \
     	rm -r /var/cache/
 
-    CMD	env | grep _TCP= | (sed 's/.*_PORT_\([0-9]*\)_TCP=tcp:\/\/\(.*\):\(.*\)/socat -t 100000000 TCP4-LISTEN:\1,fork,reuseaddr TCP4:\2:\3 \&/' && echo wait) | sh
+    CMD	env | grep _TCP= | (sed 's/.*_PORT_\([0-9]*\)_TCP=tcp:\/\/\(.*\):\(.*\)/socat -t 100000000 TCP4-LISTEN:\1,fork,reuseaddr TCP4:\2:\3 \&/' && echo wait) | sh
```
Comment (gdevillele) [inline engine/admin/ambassador_pattern_linking.md]: @thaJeztah yes, thinking about it, I should have discarded those last-line-changes before committing. Is there some documentation about this for the docs case? Are these trailing new lines required on this repo?

Comment (thaJeztah) [inline engine/admin/ambassador_pattern_linking.md]: It's general it's good practice to have the trailing newline, as it also reduces the diff if new content is added at the end

Comment (thaJeztah) [inline engine/admin/ambassador_pattern_linking.md]: No, don't think there were conflicts, just noticed these while reviewing https://github.com/docker/docker.github.io/issues/734

Comment (gdevillele) [inline engine/admin/ambassador_pattern_linking.md]: 👌

**Pull request merged**